docs/organization-integration/onpremise-integration.md

```diff
@@ -0,0 +1,183 @@
+
+## Identity provider integration
+
+Sigrid on-premise integrates with your identity provider through [OpenID Connect](https://openid.net/developers/how-connect-works/). You can either use an identity provider that "natively" supports OpenID Connect, or you can use middleware like [Dex](https://dexidp.io) or [KeyCloak](https://www.keycloak.org) that allows your identity provider to integrate with other systems.
+
+Locate the `sigrid-stack.auth-api` section in your Helm Chart's `values.yaml` file. Update the configuration properties so that they point to your identity provider. 
+
+## Development platform integration
+
+In the on-premise version of Sigrid CI, the analysis runs locally as part of your continuous integration pipeline. Importing the analysis results into Sigrid is also done within the pipeline. Unlike the software-as-a-service version, the on-premise version of Sigrid CI does *not* publish your source code to Sigrid, so your source code never leaves your development platform.
+
+The on-premise Sigrid CI requires the `sigrid-multi-analysis-import` Docker container. The steps for including this container in your pipeline are different for each development platform. The next sections cover some common examples:
+
+### GitHub pipeline integration
+
+Your can add Sigrid CI on-premise to your GitHub pipeline by creating [GitHub Actions YAML files](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) inside of the `.github/workflows` directory.
+
+```
+name: sigrid-publish
+on:
+  schedule:
+     - cron: "0 0 * * *"
+
+jobs:
+  sigridci:
+    container:
+      image: softwareimprovementgroup/sigrid-multi-analysis-import
+      env:
+        CUSTOMER: YOUR_SIGRID_CUSTOMER
+        SYSTEM: YOUR_SIGRID_SYSTEM
+        POSTGRES_HOST_AND_PORT: YOUR_DATABASE_HOST:5432
+        POSTGRES_PASS: $SIGRID_REVIEW_ONPREM_POSTGRES_IMPORTER_PASS
+    steps:
+      - ln -s $PWD /tmp/sources
+      - /usr/local/bin/all-the-things.sh
+      - mv /tmp/analysis-results/sigridci .
+```
+
+Note this is a *generic* example, the specifics might be slightly different depending on how you have configured your on-premise GitHub.
+
+This example will analyze your code on a daily basis. It is also possible to run the analysis after every change, and provide feedback in pull requests. Refer to the [Sigrid CI GitHub documentation](../sigridci-integration/github-actions.md) for more information.
+
+### GitLab pipeline integration
+
+Your can add Sigrid CI on-premise to your GitLab pipeline by editing the `.gitlab-ci.yml` configuration file:
+
+```
+sigridci-onpremise:
+  stage: test
+  image: softwareimprovementgroup/sigrid-multi-analysis-import
+  entrypoint: [""]
+  variables:
+    CUSTOMER: YOUR_SIGRID_CUSTOMER
+    SYSTEM: YOUR_SIGRID_SYSTEM
+    POSTGRES_HOST_AND_PORT: YOUR_DATABASE_HOST:5432
+    POSTGRES_PASS: $SIGRID_REVIEW_ONPREM_POSTGRES_IMPORTER_PASS
+  script:
+    - ln -s $PWD /tmp/sources
```

Comment:
Same here, maybe we can insert this in the all-the-things.sh script (without breaking if the symlink is already existing)